.rubocop.yml

```diff
@@ -24,6 +24,7 @@ Layout/LineLength:
 Metrics/AbcSize:
   Exclude:
     - "app/mailers/tigerdata_mailer.rb"
+    - "app/presenters/project_xml_presenter.rb"
```

Comment:
@jrgriffiniii I much prefer if you put the rubocop ignore into the file itself.  It shows you intended for this to happen and exactly where your intention was.  
```suggestion
```